src/containers/pages/AssigmentMapWrapper/helpers/utils.tsx

```diff
@@ -35,12 +31,8 @@ export const onJurisdictionClick = (
       deselectNodeCreator,
       currentChildren,
       plan,
-      autoSelectionFlow,
+      baseAssignmentURL,
     } = props;
-
-    const baseUrl = !autoSelectionFlow
-      ? `${MANUAL_ASSIGN_JURISDICTIONS_URL}/${plan.identifier}/${rootJurisdictionId}`
-      : `${AUTO_ASSIGN_JURISDICTIONS_URL}/${plan.identifier}/${rootJurisdictionId}`;
```

Comment:
@KipSigei would this then mean we do not need the `autoSelectionFlow` prop any more for this component.

Reply:
addressed https://github.com/onaio/reveal-frontend/pull/1161/commits/186a75e03be2bc2d8564ff1ec64b9f041f0fea44